launchcontainer/launchcontainer.py

```diff
@@ -69,11 +72,22 @@ def student_email(self):
             return None
 
     def _get_API_url(self):
-        api_conf = settings.ENV_TOKENS.get('LAUNCHCONTAINER_API_CONF', None)
-        try:
-            return api_conf[self.block_course_org]
-        except (TypeError, KeyError):
-            return API_URL_DEFAULT
+        api_conf = settings.ENV_TOKENS.get('LAUNCHCONTAINER_API_CONF', DEFAULT_API_CONF)
+        url_validator = validators.URLValidator()
+        url_validator(api_conf['HOST'])
```

Comment:
Yes, that's a good thought. I'm not sure where I would store it, though. In any case, the next PR (for microsites) will include the ability for the user to make live changes (via the Django admin), so we'll want to check on every request. Let's leave this as-is for now, then revisit on that other PR (#2).